WebDriverAgentTests/UnitTests/FBElementCacheTests.m

```diff
@@ -53,4 +53,65 @@ - (void)testResolvingFetchedElement
   XCTAssertTrue(element.didResolve);
 }
 
+- (void)testLinearCacheExpulsion
+{
+  NSMutableArray *elements = [NSMutableArray arrayWithCapacity:1050];
+  NSMutableArray *elementIds = [NSMutableArray arrayWithCapacity:1050];
+  for(int i = 0; i < 1050; i++) {
+    [elements addObject:(XCUIElement *)XCUIElementDouble.new];
+  }
+  
+  // The capacity of the cache is limited to 1024 elements. Add 1050
+  // elements and make sure:
+  // - The first 26 elements are no longer present in the cache
+  // - The remaining 1024 elements are present in the cache
+  for(int i = 0; i < 1050; i++) {
+    [elementIds addObject:[self.cache storeElement:elements[i]]];
+  }
+  
+  for(int i = 0; i < 26; i++) {
+    XCTAssertNil([self.cache elementForUUID:elementIds[i]]);
+  }
+  for(int i = 27; i < 1050; i++) {
```

Comment:
Please add expressions where you calculate these numbers, so it would be visible where we get them.

Reply:
Done, thx.